src/js/components/ServiceSidebarFilters.js

```diff
@@ -20,6 +20,10 @@ function getCountByFilters(services) {
   const volumesKey = ServiceOther.VOLUMES.key;
 
   return services.reduce(function (memo, service) {
+    if (service instanceof ServiceTree) {
+      return memo;
+    }
```

Comment:
we do not want to count `groups` into the filter count at all. this is because when we use sidebar filtering, we do not show groups